submissions/benjaminKamena/aboutMe/aboutMe.txt.txt

```diff
@@ -0,0 +1,8 @@
+Am benjamin kamena, 
```

Comment:
@kamena1994 This file exist already. Remove it from your pull request. To do that, make sure you are on this branch, delete this file, do `git add .`, commit the changes, then push.